firedantic/tests/tests_sync/test_model.py

```diff
@@ -89,6 +89,49 @@ def test_find(configure_db, create_company, create_product):
         Product.find({"product_id": {"<>": "a"}})
 
 
+def test_find_not_in(configure_db, create_company):
+    ids = ["1234555-1", "1234567-8", "2131232-4", "4124432-4"]
+    companies = []
+    for company_id in ids:
+        companies.append(create_company(company_id=company_id))
+
+    found: List[Company] = Company.find(
+        {
+            "company_id": {
+                "not-in": [
+                    "1234555-1",
+                    "1234567-8",
+                ]
+            }
+        }
+    )
+    assert len(found) == 2
+    for company in found:
+        assert company.company_id in ("2131232-4", "4124432-4")
+
+
+def test_find_array_contains(configure_db, create_todolist):
+    list_1 = create_todolist("list_1", ["Work", "Eat", "Sleep"])
+    create_todolist("list_2", ["Learn Python", "Walk the dog"])
+
+    found: List[TodoList] = TodoList.find({"items": {"array_contains": "Eat"}})
```

Comment:
So how about creating `op.py` file and define all the supported operations there? e.g:
```
ARRAY_CONTAINS = "array_contains"
NOT_IN = "not-in"
LT = "<"
LTE = "<="
```
It will allow to avoid the confusion with such inconsistencies. 
Then you could write just:
```
TodoList.find({"items": {op.ARRAY_CONTAINS: "Eat"}})
# or 
TodoList.find({"items": {ARRAY_CONTAINS: "Eat"}})
```

Reply:
I liked the idea. However, I prefer to call it `operators.py`; more understandable name, you can still import `as op`.